packages/ontologies/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@canonical/ontologies",
+  "description": "DS Ontologies",
+  "version": "0.10.0-experimental.4",
+  "type": "module",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist",
+    "icons"
+  ],
```

Comment:
Update, this is targeting the `ds-assets` file exports

Also consider marking the package as `private` if accurate